www/polj10/sp1/index.html

```diff
@@ -0,0 +1,397 @@
+                        <td><a href="https://www.rvvi.cz/cep?s=jednoduche-vyhledavani&ss=detail&n=0&h=LM2015081">LM2015081</a></td>
+                    </tr>
+                    <tr>
+                        <td>Akronym výzkumné infrastruktury:</td>
+                        <td><abbr title="Research Infrastructure for Diachronic Czech Studies">RIDICS</abbr></td>
+                    </tr>
+                    <tr>
+                        <td>Výzkumné oblasti:</td>
+                        <td>společenské a humanitní vědy (hlavní), informační a komunikační technologie /
+                            e-infrastruktury (vedlejší)</td>
+                    </tr>
+                    <tr>
+                        <td>Hostitelská instituce:</td>
+                        <td><a href="http://www.ujc.cas.cz/" target="_blank">Ústav pro jazyk český AV ČR, v. v. i.</a></td>
+                    </tr>
+                    <tr>
+                        <td>Statutární orgán:</td>
+                        <td>PhDr. Martin Prošek, Ph.D.</td>
+                    </tr>
+                    <tr>
+                        <td>Partnerská instituce:</td>
+                        <td><a href="http://www.fel.cvut.cz">České vysoké učení technické v Praze, Fakulta
+                                elektrotechnická</a></td>
+                    </tr>
+                    <tr>
+                        <td>Odpovědná osoba:</td>
+                        <td>Dalibor Lehečka</td>
+                    </tr>
+                    <tr>
+                        <td>Kontakt:</td>
+                        <td><a href="mailto:vokabular@ujc.cas.cz">vokabular@ujc.cas.cz</a></td>
+                    </tr>
+                </table>
+            </div>
+            <h2>Podmínky přístupu</h2>
+            <p>
+                Data a nástroje výzkumné infrastruktury jsou přístupné zdarma všem badatelům.
+            </p>
+            <p>
+                Uživatel se zavazuje, že na zdroje, popř. nástroje získané prostřednictvím webové aplikace, odkáže v
+                každé publikaci, kvalifikační, doktorské nebo habilitační práci, a to následující doporučenou formou:
+                „Při vzniku práce byly využity zdroje Výzkumné infrastruktury pro diachronní bohemistiku (RIDICS,
+                http://ridics.ujc.cas.cz).“
+            </p>
+            <p>
+                Při citaci konkrétního zdroje, který je v rámci RIDICS dostupný, platí obvyklá pravidla pro citaci
+                webových stránek a příspěvků.
+            </p>
+
+            <h2>Popis zaměření výzkumné infrastruktury</h2>
+            <p>
+                Výzkumná infrastruktura pro diachronní bohemistiku (akronym RIDICS) bude obsahově spoluvytvářet a
+                spravovat dva vzájemně komplementární webové portály umožňující a podněcující výzkum v oblasti
+                diachronní bohemistiky (tj. češtiny od nejstarších období do konce 18. století) a souvisejících oborů.
```

Comment:
dlouhe texty se blbe ctou, vloz tam nejaky obrazek, pripadne tex dozdel do sloupcu, muzes pouzit css vlastnost column-count

Reply:
Dva sloupce mi nepřijdou v tomhle layoutu hezký. Obrázky bych tam mohl dát, ale bylo by to "nasilu", protože v zadání věděckýho projektu obrázky prostě nejsou.